include/boost/math/policies/policy.hpp

```diff
@@ -132,7 +132,7 @@ namespace policies{
 
 #define BOOST_MATH_META_INT(Type, name, Default)                                                \
    template <Type N = Default>                                                                  \
-   class name : public boost::math::integral_constant<int, N>{};                                \
+   class name : public boost::math::integral_constant<int, static_cast<int>(N)>{};              \
```

Comment:
Should we cast to `Type` here since the argument could be a short or something (I'm sure it never is in practice, but it could be).

Reply:
I think you're right. But to be clear, you mean changing to something like below, right? That would make the most sense to me upon reading your comment.

```cpp
class name : public boost::math::integral_constant<Type, N>{};
```

Reply:
Yes.  But thinking about it, I'm not sure if that change might have unintended ripple effects - you might need to try it and see!

Reply:
Hi John. It worked fine. I had a few typos underway. I will now merge this.

Cc: @jzmaddock and @mborland